.github/workflows/feature.yaml

```diff
@@ -33,7 +33,6 @@ jobs:
       fail-fast: false
       matrix:
         node_js_version:
-          - '14.17.0'
           - '16'
           - '18'
```

Comment:
Could you also add '20' here?

Reply:
Sure! I also updated the version for other CI jobs.